tests/backup/backup_kubevirt_test.go

```diff
@@ -1335,3 +1335,217 @@ var _ = Describe("{KubevirtVMBackupRestoreWithNodeSelector}", func() {
 		CleanupCloudSettingsAndClusters(backupLocationMap, cloudCredName, cloudCredUID, ctx)
 	})
 })
+
+var _ = Describe("{KubevirtVMWithFreezeUnfreeze}", func() {
+	var (
+		//restoreNames         []string
+		scheduledAppContexts []*scheduler.Context
+		sourceClusterUid     string
+		cloudCredName        string
+		cloudCredUID         string
+		backupLocationUID    string
+		backupLocationName   string
+		backupLocationMap    map[string]string
+		providers            []string
+		labelSelectors       map[string]string
+		//controlChannel       chan string
```

Comment:
Can you move all the commented code in a single block and add a TODO on top? So that we don't lose track?

Reply:
Makes sense